### ISSUE TITLE
DI exploit used to remove the 10% dmg reduction from titan's grip talent

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5633]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If you Divine Intervention a fury warrior he will lose the 10% dmg reduction from titans grip talent same as before hand of protection was removing the dmg reduction
**Expected behaviour**: Tell us what should happen instead
Using DI shouldn't remove the dmg reduction from titans grip
**Steps to reproduce the problem**:

1. log a fury warr check ur current dmg
2. ask a pala to DI you
3. check again how the dmg increased

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by MalkaviaDW** [comment]
#4365

**Comment by RandomXD** [comment]
still an issue and its being exploited
